src/app/lib/utilities/logging/pathWithLogging/index.js

```diff
@@ -19,15 +18,20 @@ const getLoggingFunction = level =>
     [LOG_LEVELS.ERROR]: logger.error,
   }[level] || logger.info);
 
-export default (fieldPath, { logLevel = LOG_LEVELS.INFO } = {}) => pageData => {
+const pathWithLogging = (url, logCategory, pageData) => (
```

Comment:
Can we maybe add some docs here to describe what this function is doing?

Reply:
Added readme.md